packages/create-nx-workspace/package.json

```diff
@@ -27,8 +27,6 @@
   "bugs": {
     "url": "https://github.com/nrwl/nx/issues"
   },
-  "main": "./index.js",
-  "typings": "./index.d.ts",
```

Comment:
Why remove these?

Reply:
it will be automatically generated on the build step, it is not needed since main for build target is set to index.ts